src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj

```diff
@@ -3,32 +3,48 @@
   <PropertyGroup>
     <TargetFramework>net452</TargetFramework>
     <AssemblyName>NServiceBus.Transports.RabbitMQ</AssemblyName>
-    <Copyright>Copyright NServiceBus. All rights reserved</Copyright>
-    <Company>NServiceBus Ltd.</Company>
+    <Product>NServiceBus.RabbitMQ</Product>
+    <AssemblyTitle>NServiceBus.RabbitMQ</AssemblyTitle>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\NServiceBus.snk</AssemblyOriginatorKeyFile>
     <OutputPath>..\..\binaries\</OutputPath>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
-    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
```

Comment:
GitVersion 3.x did not know how to work with the new project system, so it was conflicting by generating duplicate properties. GitVersion 4.x knows how to fill its information into the right spots, so we can let the project generate these values again.